src/app/features/discarded-inscriptions/use-inscribed-spendable-utxos.ts

```diff
@@ -0,0 +1,49 @@
+import { useMemo } from 'react';
+
+import { useNativeSegwitUtxosByAddress } from '@leather.io/query';
+
+import { useCurrentNativeSegwitInscriptions } from '@app/query/bitcoin/ordinals/inscriptions/inscriptions.query';
+import { useCurrentAccountNativeSegwitIndexZeroSignerNullable } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+import { useDiscardedInscriptions } from '@app/store/settings/settings.selectors';
+
+export function useInscribedSpendableUtxos() {
+  const { hasInscriptionBeenDiscarded } = useDiscardedInscriptions();
+
+  const { data: nativeSegwitInscriptions } = useCurrentNativeSegwitInscriptions();
+
+  const nativeSegwitSigner = useCurrentAccountNativeSegwitIndexZeroSignerNullable();
+  const address = nativeSegwitSigner?.address;
+
+  // Utxos but don't filter the inscribed ones
+  const { data: nativeSegwitUtxos } = useNativeSegwitUtxosByAddress({
+    address: address ?? '',
```

Comment:
Totally agree and only did this as I'm trying to get it done quickly before Christmas.

The way I see it, resorting to empty strings here (as done in many other places), is symptomatic of poor patterns around how we pass and handle state.

I've been thinking about this with this work. One solution would be to use feature container components to handle null cases, and set state in contexts state such that it can be consumed without the need to inline null fallbacks in every case.